test.js

```diff
@@ -15,6 +15,8 @@ const {
   hash,
   createError,
 } = require('./src');
+const { setHttpCacheHeaders } = require('./src/middleware');
+const { request } = require('http');
```

Comment:
Where is this use?

Reply:
Line 18 is the import for my module, I don't use Line 19, I'll remove that